torch/csrc/autograd/custom_function.cpp

```diff
@@ -3,12 +3,19 @@
 
 namespace torch { namespace autograd {
 
-std::vector<Variable> _wrap_outputs(const std::unordered_set<at::TensorImpl*> &inputs,
+variable_list _wrap_outputs(const variable_list &input_vars,
```

Comment:
OK :) This seems like it might be a reasonable change to land ahead of time before all of the custom autograd functions?